### PR TITLE
Updated project bootstrap.datetimepicker to 4.15.35

### PR DESCRIPTION
**Pull request by evorios**
Updated project bootstrap.datetimepicker to 4.15.35

### CODE REVIEW

File: files/bootstrap.datetimepicker/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "bootstrap-datetimepicker",
+  "repo": "Eonasdan/bootstrap-datetimepicker",
+  "files": {
+    "basePath": "build/"
```
Comment (jimaek) [inline files/bootstrap.datetimepicker/update.json]: Please add the include parameter to use only the min files.

Comment (evorios) [inline files/bootstrap.datetimepicker/update.json]: Fixed.

File: files/bootstrap.datetimepicker/update.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "bootstrap-datetimepicker",
+  "repo": "Eonasdan/bootstrap-datetimepicker",
+  "files": {
+    "basePath": "build/",
+    "include": ["css/bootstrap-datetimepicker.min.css", "js/bootstrap-datetimepicker.min.js"]
```
Comment (jimaek) [inline files/bootstrap.datetimepicker/update.json]: Please add `./` before each file. This way the path will be retained.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
The instruction for that is here https://github.com/jsdelivr/libgrabber#add-updatejson-schema
>dist/*.js - copies javascript files from dist dir. Directory structure will not be retained (e.g. dist dir will be stripped when copied). To retain directory structure prepend glob with ./ e.g. ./vendors/*.js 

I doubt Habr will be interested to learn how to add projects here, it would basically a documentation.

**Comment by evorios** [comment]
@jimaek , What mean "name" in config file update.json?
I wrote `"name": "bootstrap-datetimepicker"`, it's name of itself github repo.
But in [other project](https://github.com/jsdelivr/jsdelivr/blob/master/files/jquery.validation.unobtrusive/update.json) written `"name": "jquery.validation.unobtrusive"`, it's name of project in jsdelivr repo.

**Comment by jimaek** [comment]
Name is required by npm and bower, its used to download the project.
For Github you can enter anything you want, its used in the logs

**Pull request closed without merging**